format/spec.md

```diff
@@ -950,6 +977,7 @@ Maps with non-string keys must use an array representation with the `map` logica
 |**`uuid`**|`{ "type": "fixed",`<br />&nbsp;&nbsp;`"size": 16,`<br />&nbsp;&nbsp;`"logicalType": "uuid" }`||
 |**`fixed(L)`**|`{ "type": "fixed",`<br />&nbsp;&nbsp;`"size": L }`||
 |**`binary`**|`bytes`||
+|**`variant`**|`bytes` containing concatenated metadata and value||
```

Comment:
Would this be a special record similar to `group` for Parquet?

Reply:
@aihuaxu, yes. I think this should be a record of two binary values, the metadata and value. I should update this.

Reply:
I've removed variant so that we can move forward with this PR separately.